docs/examples/ex_extract.rst

```diff
@@ -0,0 +1,669 @@
+      * lev           (lev) float64 -7.0 -6.75 -6.5 -6.25 -6.0 ... 6.25 6.5 6.75 7.0
+      * ilev          (ilev) float64 -6.875 -6.625 -6.375 ... 6.625 6.875 7.125
+    Data variables:
+        uts           (time) float64 0.0 3.6e+03 7.2e+03 ... 7.92e+04 8.28e+04
+        altitude      (time, ilev, latitude, longitude) float64 0.0 0.0 ... 5.84e+07
+        dummy_drifts  (time, ilev, latitude, longitude) float64 0.0 0.0 ... 83.01
+        slt           (time, longitude) float64 12.0 12.17 12.33 ... 10.67 10.83
+        mlt           (time, longitude) float64 12.2 12.37 12.53 ... 10.87 11.03
+        dummy1        (time, latitude, longitude) float64 0.0 0.0 0.0 ... 0.0 9.0
+
+Model variables, such as ``dummy_drifts``, are regular over
+``(time, ilev, latitude, longitude)``, where ``ilev`` is a constant
+pressure level. Unfortunately, the observational data in ``inst`` doesn't
+contain pressure level as a simulated/measured parameter. However,
+``altitude`` is present in the model data but varies over all four
+coordinates. Interpolating ``dummy_drifts`` onto ``inst`` requires
+either adding an appropriate value for ``ilev`` into ``inst``, or
+interpolating model variables using the irregular variable ``altitude``
+instead of ``ilev``.
+
+Altitude to Pressure
+^^^^^^^^^^^^^^^^^^^^
+
+:py:func:`pysatModels.utils.extract.instrument_altitude_to_model_pressure`
+will use information in a model to generate appropriate pressure levels for a
+supplied altitude in an observational-like data set.
+
+.. code:: python
+
+    import pysatModels
+
+    keys = pysatModels.utils.extract.instrument_altitude_to_model_pressure(inst,
+                model.data, ["altitude", "latitude", "longitude"],
+                ["ilev", "latitude", "longitude"],
+                "time", "time", ['', "deg", "deg"],
+                'altitude', 'altitude', 'cm')
+
+The function will guess a pressure level for all locations in ``inst``
+and then use the regular mapping from pressure to altitude to obtain the
+equivalent altitude from the model. The pressure is adjusted up/down an
+increment based upon the comparison and the process is repeated until the
+target tolerance (default is 1 km) is achieved. The keys for the model derived
+pressure and altitude values added to ``inst`` are returned from the
+function.
+
+.. code:: python
+
+    inst['model_pressure']
+
+    Epoch
+    2009-01-01 00:00:00    3.104662
+    2009-01-01 00:00:01    3.104652
+    2009-01-01 00:00:02    3.104642
+    2009-01-01 00:00:03    3.104632
+    2009-01-01 00:00:04    3.104623
+                             ...
+    2009-01-01 23:59:55    2.494845
+    2009-01-01 23:59:56    2.494828
+    2009-01-01 23:59:57    2.494811
+    2009-01-01 23:59:58    2.494794
+    2009-01-01 23:59:59    2.494776
+    Name: model_pressure, Length: 86400, dtype: float64
+
+    # Calculate difference between interpolation techniques
+    inst['model_altitude'] - inst['altitude']
```

Comment:
Maybe a break in the formatting boxes to separate the output of the previous call from the next command?